dashboard/app/views/layouts/application.html.haml

```diff
@@ -86,6 +86,9 @@
           = render partial: 'layouts/small_footer', locals: {full_width: true}
 
     - unless view_options[:no_footer]
+      - if view_options[:above_footer_content]
+        = render partial: view_options[:above_footer_content]
```

Comment:
This would be a great place to use the [`content_for` method](https://edgeguides.rubyonrails.org/layouts_and_rendering.html#using-the-content-for-method) ([docs](https://api.rubyonrails.org/classes/ActionView/Helpers/CaptureHelper.html#method-i-content_for))

Reply:
![giphy (7)](https://user-images.githubusercontent.com/224089/64884667-2089bb80-d617-11e9-93a4-b4be4dfc96d0.gif)